apps/docs/components/hooks/useScrollable.md

```diff
@@ -81,6 +80,71 @@ const { getContainerRef, state, getNextButtonProps, getPrevButtonProps } = useSc
 ```
 </SourceCode>
 <!-- end vue -->
+
+### Controlled showing next and previous elements
+<!-- react -->
+
+`useScrollable` is exposing imperative methods for showing elements - `showNext` and `showPrev`. To make them work, user needs to bind these functions as `onClick` handler.
+
+<SourceCode>
+
+```tsx
+import * as React from 'react';
+import { useScrollable } from '@storefront-ui/react';
+
+function BasicScrollable(props: Props) {
+  const { containerRef, state, showNext, showPrev } = useScrollable();
+
+  return (
+    <div className="flex items-center">
+      <button onClick={showPrev()}>previous</button>
```

Comment:
you could also apply `disabled={!state.hasPrev}` and `disabled={!state.hasNext}` to reflect the same behaviour as in the props getters.